projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -348,14 +348,14 @@
 		color: var(--jetpack--contact-form--text-color);
 		display: flex;
 		border: none;
```

Comment:
Not important to the PR, but I'm curious why the CSS selector for this rule is duplicated (`.jetpack-option__type.jetpack-option__type`). I assume that's just for greater specificity.

Reply:
Yes, repeating the selector increases the specificity. I assume that this makes it possible to apply some style property.